specification/dimensions/usageunit.md

```diff
@@ -0,0 +1,93 @@
+# Usage Unit
+
+Usage Unit is the unit of measure for the usage or consumption of a resource or service. Usage Unit is used in conjunction with Quantity In Usage Unit to gain understanding of usage or consumption of a given product or service in an environment.
+
+The UsageUnit column MUST be present in the billing data. This column MUST be of type String and MUST NOT contain null values when the ChargeType is 'Usage'. UsageUnit should be expressed as a single unit of measure.  Units of measure used in UsageUnit SHOULD adhere to the values and format requirements specified in the Recommended Values section below.
```

Comment:
I think here we have to go MUST and then in the specific requirements, we have some SHOULDs, some MUSTs and some MAYs. If we go SHOULD here, a MUST in the below section becomes optional? This is the first time we're running into this so check my logic and accept only if it makes sense.

```suggestion
The UsageUnit column MUST be present in the billing data. This column MUST be of type String and MUST NOT contain null values when the ChargeType is 'Usage'. UsageUnit should be expressed as a single unit of measure.  Units of measure used in UsageUnit MUST adhere to the values and format requirements specified in the Recommended Values section below.
```